src/app.module.ts

```
@@ -101,4 +101,32 @@ import { PathwaysModule } from './pathways/pathways.module';
     },
   ],
 })
-export class AppModule {}
+export class AppModule implements OnModuleInit {
+  private readonly logger = new Logger(AppModule.name);
+
+  onModuleInit() {
+    this.logger.log('AppModule initialized successfully');
```

ESLint

[error] Replace `'AppModule·initialized·successfully'` with `"AppModule·initialized·successfully"`

(prettier/prettier)